src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -168,3 +169,57 @@ def test_3rd_body_Curtis(test_params):
                               (argp_f * u.rad).to(u.deg) - test_params['orbit'][4]],
                              [test_params['raan'], test_params['inc'], test_params['argp']],
                              rtol=1e-1)
+
+
+solar_pressure_checks = [{'t_days': 200, 'deltas_expected': [3e-3, -8e-3, -0.035, -80.0]},
+                         {'t_days': 400, 'deltas_expected': [-1.3e-3, 0.01, -0.07, 8.0]},
+                         {'t_days': 600, 'deltas_expected': [7e-3, 0.03, -0.10, -80.0]},
+                         {'t_days': 800, 'deltas_expected': [-7.5e-3, 0.02, -0.13, 1.7]},
+                         {'t_days': 1000, 'deltas_expected': [6e-3, 0.065, -0.165, -70.0]},
+                         {'t_days': 1095, 'deltas_expected': [0.0, 0.06, -0.165, -10.0]},
+                         ]
+
+
+def normalize_to_Curtis(t0, sun_r):
+    r = sun_r(t0)
+    return 149600000 * r / norm(r)
+
+
+def test_solar_pressure():
+    # based on example 12.9 from Howard Curtis
+    solar_system_ephemeris.set("jpl")
+
+    j_date = 2438400.5
+    tof = 1095
+    sun_r = build_ephem_interpolant(Sun, 365, (j_date, j_date + tof), rtol=1e-2)
+    epoch = Time(j_date, format='jd', scale='tdb')
+
+    a_ini = 10085.44 * u.km
+    ecc_ini = 0.025422 * u.one
+    inc_ini = 88.3924 * u.deg
+    raan_ini = 45.38124 * u.deg
+    argp_ini = 227.493 * u.deg
+    nu_ini = 343.4268 * u.deg
+    initial = Orbit.from_classical(Earth, a_ini, ecc_ini, inc_ini, raan_ini, argp_ini, nu_ini, epoch=epoch)
+    # in Curtis, the mean distance to Sun is used. In order to validate against it, we have to do the same thing
+    sun_normalized = functools.partial(normalize_to_Curtis, sun_r=sun_r)
+
+    r, v = cowell(initial, np.linspace(0, (tof * u.day).to(u.s).value, 4000), rtol=1e-8, ad=solar_pressure,
+                  R_Earth=Earth.R.to(u.km).value, C_R=2.0, A=2e-4, m=100, Wdivc_s=Sun.Wdivc.value, sun=sun_normalized)
+
+    delta_as, delta_eccs, delta_incs, delta_raans, delta_argps, delta_hs = [], [], [], [], [], []
+    for ri, vi in zip(r, v):
```

Comment:
Wow, so this is the required code to get a time history of orbital elements... There should be an easier way to do it. Not for this pull request, of course.